src/DssSpell.t.base.sol

```diff
@@ -200,10 +200,10 @@ contract DssSpellTestBase is Config, DSTest, DSMath {
         // Test for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0x3142dafd5aD998E35EB8E4Bc8E7D01545E02281A),        // populate with deployed spell if deployed
-            deployed_spell_created:         1643391294,        // use get-created-timestamp.sh if deployed
+            deployed_spell:                 address(0xC45c1419DA490880b4cf4e840e14D62827969B3f),        // populate with deployed spell if deployed
+            deployed_spell_created:         1644002846,        // use get-created-timestamp.sh if deployed
             previous_spell:                 address(0),        // supply if there is a need to test prior to its cast() function being called on-chain.
-            office_hours_enabled:           false,             // true if officehours is expected to be enabled in the spell
+            office_hours_enabled:           true,              // true if officehours is expected to be enabled in the spell
             expiration_threshold:           weekly_expiration  // (weekly_expiration,monthly_expiration) if weekly or monthly spell
         });
         spellValues.deployed_spell_created = spellValues.deployed_spell != address(0) ? spellValues.deployed_spell_created : block.timestamp;
```

Comment:
Missing `esm_min` in base tests and config.
Check changes to apply in [PE-838 spells-goerli](https://github.com/makerdao/spells-goerli/pull/42), [Goerli-DssSpell.t.base.sol](https://github.com/makerdao/spells-goerli/pull/42/commits/a4ca2571a5d1a708e60871e1add747a35fd1fbd3#diff-66a33be4e15ec87add1b15eb9fe8e292c82f68ffe873e045f2bda0588b2fea1c) and [config.sol](https://github.com/makerdao/spells-goerli/pull/42/commits/a4ca2571a5d1a708e60871e1add747a35fd1fbd3#diff-15b651e461207738a0b33018c6baac15bba72e0332384b82fa69871f27601f6e)

Reply:
We should add these before archive. It shouldn't block handoff since we have these as requires in the spell.

Reply:
Done.